### ISSUE TITLE
favicon for the site? (image in browser tab)

### ISSUE THREAD

**Issue by ebeshero** [Samantha-Mcguigan/newsAnalysis#20]
I may not have told you all about favicons, and setting one up on your site...the other projects from last term have them... It's the little tiny image that shows in the browser tab. You can set that by choosing an image (any image) that you think would look sharp when shrunk to like 16 x 16 pixels, and save it as favicon.ico in the root of your project directory.

<img width="380" alt="screenshot 2017-04-24 21 59 12" src="https://cloud.githubusercontent.com/assets/4014518/25365945/3f445936-293a-11e7-8458-42903bbe4674.png">

**Comment by dotfig** [comment]
I put one in the server but nothing is showing up

**Comment by ebeshero** [comment]
Try adding this line to your pages: `<link rel="icon" href="favicon.ico" type="image/x-icon" />` 

Looks like that was necessary for the project teams last year...Also, I don't think Safari shows them, but the other browsers do (Firefox, for example).

**Comment by ebeshero** [comment]
@ajnewton1 I'm reading that you may need to clear your browser's cache (local stored version) of your pages to see a new favicon (and that is what you need to view a change in CSS or JavaScript as well). Also you can make sure it's available by calling it up in the browser directly, with 
http://newsanalysis.newtfire.org/favicon.ico
And then going back and refreshing the page that you want to show it. (Looks like the file isn't there when I follow the link.)
Apparently Safari does render these, but maybe you have to set up/configure the browser display for it. Anyway, it's just a neat cosmetic feature. :-)

**Comment by ahunker** [comment]
@ajnewton1 
@ebeshero  Was telling me you were having trouble with the favicon and I was able to get ours to work. I have a Mac as well and Safari, in general, doesn't show the favicon but when I opened the page in GoogleChrome, the icon showed up. I just relabelled an image, put it in the SSI, and popped the above link in the head of my index page and the favicon showed up on all my pages. Hope this helps you out.
